asserts/model.go

```diff
@@ -341,6 +341,26 @@ const (
 
 var validModelGrades = []string{string(ModelSecured), string(ModelSigned), string(ModelDangerous)}
 
+// gradeToCode encodes grades into 32 bits, trying to be slightly future-proof:
+// * lower 16 bits are reversed
```

Comment:
Maybe this should be "reserved"? Otherwise I didn't understand what's intended here.

Reply:
yes, oops, what I meant is reserved